man/org.freedesktop.login1.xml

```diff
@@ -738,11 +739,10 @@ node /org/freedesktop/login1 {
       should be a short human readable string identifying the reason why the lock is taken. Finally,
       <varname>mode</varname> is either <literal>block</literal> or <literal>delay</literal> which encodes
       whether the inhibit shall be consider mandatory or whether it should just delay the operation to a
-      certain maximum time, while the <literal>block-weak</literal> and <literal>delay-weak</literal>
-      variants will create an inhibitor that is automatically ignored in some circumstances. The method
-      returns a file descriptor. The lock is released the moment this file descriptor and all its duplicates
-      are closed. For more information on the inhibition logic see
-      <ulink url="https://systemd.io/INHIBITOR_LOCKS">Inhibitor Locks</ulink>.
+      certain maximum time, while the <literal>block-weak</literal> and variants will create an inhibitor
+      that is automatically ignored in some circumstances. The method returns a file descriptor. The lock is
```

Comment:
not addressed: "some circumstances"

Reply:
Yep - I actually don't really know for sure what the all the circumstances are, which is part of the problem with the original behaviour. In case you do know the exact details, please file a follow-up PR to make it more precise.

Reply:
please, when making modifications to some logic try to understand the old logic first. just commiting shit docs like this is not acceptable.